docs/src/user-guide/quick-start/clp-json.md

```diff
@@ -0,0 +1,175 @@
+# clp-json quick-start
+
+This page will walk you through how to start CLP and use it to compress and search JSON logs.
+
+:::{caution}
+If you're using a `clp-json` release, you can only compress and search JSON logs. This limitation
+will be addressed in a future version of CLP.
+:::
+
+---
+
+## Starting CLP
+
+To start CLP, run:
+
+```bash
+sbin/start-clp.sh
+```
+
+:::{note}
+If CLP fails to start (e.g., due to a port conflict), try adjusting the settings in
+`etc/clp-config.yml` and then run the start command again.
+:::
+
+---
+
+## Compressing JSON logs
+
+To compress some JSON logs, run:
+
+```bash
+sbin/compress.sh --timestamp-key '<timestamp-key>' <path1> [<path2> ...]
+```
+
+* `<timestamp-key>` is the field path of the kv-pair that contains the timestamp in each log event.
+  * E.g., if your log events look like
+    `{"timestamp": {"iso8601": "2024-01-01 00:01:02.345", ...}}`, you should enter
+    `timestamp.iso8601` as the timestamp key.
+
+  :::{caution}
+  Log events without the specified timestamp key will *not* have an assigned timestamp. Currently,
+  these events can only be searched from the command line (when you don't specify a timestamp
+  filter).
+  :::
+
+* `<path...>` are paths to JSON log files or directories containing such files.
+  * Each JSON log file should contain each log event as a
+    [separate JSON object](./index.md#clp-json), i.e., *not* as an array.
+
+The compression script will output the compression ratio of each dataset you compress, or you can
+use the UI to view overall statistics.
+
+Compressed logs will be stored in the directory specified by the `archive_output.storage.directory`
+config option in `etc/clp-config.yml` (`archive_output.storage.directory` defaults to
+`var/data/archives`).
+
+:::{tip}
+To compress logs from object storage, see
+[Using object storage](../guides-using-object-storage/index).
+:::
+
+### Sample logs
+
+For some sample logs, check out the open-source [datasets](../resources-datasets).
+
+---
+
+## Searching JSON logs
+
+You can search your compressed logs from CLP's [UI](#searching-from-the-ui) or the
+[command line](#searching-from-the-command-line).
+
+Queries must be in the form of a set of conditions (predicates) on key-value pairs (kv-pairs). For
+example, consider the logs in [Figure 1](#figure-1) and the query in [Figure 2](#figure-2).
```

Comment:
```suggestion
In clp-json, queries are written as a set of conditions (predicates) on key-value pairs (kv-pairs).
For example, [Figure 1](#figure-1) shows a query that matches the first log event in
[Figure 2](#figure-2).
```

Edited based on your writing in clp-text.md. Can you swap the content of the figures?